docs/GITOPS-CONFIG.md

```diff
@@ -20,31 +20,43 @@ If you installed using the [`ai-rhdh-installer`](../README.md#install) all that
 ### AI-RHDH-Installer: Manual Configuration
 
 #### Step 1: ConfigMaps and Secrets
-In your chosen namespace you should apply the following ConfigMaps:
-1.  [`argocd-config.yaml` ](../resources/argocd-config.yaml)
-2.  [`argocd-plugins.yaml`](../resources/argocd-plugins.yaml)
+In your chosen namespace you should apply the `argocd-config` ConfigMap under [`argocd-config.yaml` ](../resources/argocd-config.yaml).
 
 As part of the `ai-rhdh-installer` a secret was created in your desired namespace with the name `rhdh-argocd-secret`, keep note of this Secret as well as the two ConfigMaps applied above.
 
 #### Step 2: Updating Plugins
 **Note: RHDH will encounter errors if the ArgoCD plugins are loaded before an instance is properly attached - This step may result in errors until all steps are completed**
 
-To include the ArgoCD plugins we need to edit the dynamic plugins ConfigMap that was created by the RHDH Operator:
+To include the [ArgoCD plugins list](../dynamic-plugins/argocd-plugins.yaml) we need to edit the dynamic plugins ConfigMap that was created by the RHDH Operator:
 
 ![Dynamic Plugins Example](../assets/dynamic-plugins-example.png)
 
-Edit the associated `yaml` file to include `argocd-plugins.yaml` under the `includes` section:
+Edit the associated `yaml` file to include the contents of the [ArgoCD plugins list](../dynamic-plugins/argocd-plugins.yaml) under the `plugins` section:
 
 ![Dynamic Plugins Example 2](../assets/dynamic-plugins-example-2.png)
 
-#### Step 3: Updating RHDH Deployment
-Now that all of the required ConfigMaps and Secrets are apart of the namespace, as well as adding the plugins we must edit the RHDH Deployment to properly reference these items.
-
-Since we are referencing `argocd-plugins.yaml` in our dynamic-plugins ConfigMap we first must add `argocd-plugins.yaml` as a `volumeMount` to the `initContainers` field in the RHDH Deployment under `.spec.template.spec.initContainers.volumeMounts`
+Alternatively, we can use this series of commands to perform the same task with `kubectl` and `yq` using the [`argocd-plugins.yaml`](../dynamic-plugins/argocd-plugins.yaml):
+
+1. Fetch the dynamic plugins ConfigMap and save the `dynamic-plugins.yaml` content within to a temp file
+    ```sh
+    kubectl get configmap backstage-dynamic-plugins-ai-rh-developer-hub -n ai-rhdh -o yaml | yq '.data["dynamic-plugins.yaml"]' > temp-dynamic-plugins.yaml
```

Comment:
The namespace might defer from `ai-rhdh` right? So it might be better here to include something like `<your-namespace>`?

Reply:
Yes I believe even in the ai-rhdh-installer case you can change the namespace you deploy the chart too so for clarity we should probably make this `<namespace>` instead of a hard `ai-rhdh`

Reply:
@Jdubrick @thepetk I decided to use the env var `$NAMESPACE` that refers to the rhdh namespace https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/7/commits/b8753424110732cc94a6f2aec5575f72240e2392 as this step of setting it is highlighted at the beginning of each of these documents:
- [PIPELINES-CONFIG.md](https://github.com/michael-valdron/ai-rhdh-installer/blob/b8753424110732cc94a6f2aec5575f72240e2392/docs/PIPELINES-CONFIG.md?plain=1#L16)
- [GITOPS-CONFIG.md](https://github.com/michael-valdron/ai-rhdh-installer/blob/b8753424110732cc94a6f2aec5575f72240e2392/docs/GITOPS-CONFIG.md?plain=1#L19)

Reply:
sgtm